### ISSUE TITLE
Add theme support for drush en (enable) and drush dis (disable)

### ISSUE THREAD

**Issue by jackaponte** [backdrop-contrib/drush#188]
While `drush en` and `drush dis` both work well for modules, neither works for themes.

Testing with Backdrop 1.13.2, Drush 8.2.3 and Backdrop Drush commands 1.x-1.x (most recent commit).

Results of running `drush dis basis` when the Basis theme is visible as enabled in both Drush and the Backdrop UI:

```
drush dis basis
The following projects will be disabled: basis.
    Do you want to disable the projects? (y/n): y

	Success: basis are disabled.
```

However both `drush pml` and the UI show that the Basis theme is still enabled.

Results of running `drush en basis`, whether or not Basis is actually disabled:

```
drush en basis        
The following projects will be enabled: basis.
    Do you want to enable the projects? (y/n): y

	Error basis does not exist in your Backdrop installation.
	Try downloading basis first with the command: drush dl basis
```

This happens with the Basis and Bartik core themes as well as with a custom theme.

**Comment by serundeputy** [comment]
You are correct! On the initial pass of `en/dis` commands no attempt was made to encompass themes and layouts.  With the intention of opening a follow up issue for those features.  You did that here!

Let's use this issue for `en/dis` support for themes.

**Comment by jackaponte** [comment]
> You are correct! On the initial pass of `en/dis` commands no attempt was made to encompass themes and layouts. With the intention of opening a follow up issue for those features. You did that here!

Ah great! :)

> Let's use this issue for `en/dis` support for themes.

Sounds good. I'm going to give this a look and see if it's something I think I can sort out, at least partially; assigning this to myself for that.
